packages/slice-machine/tests/project/cypress/integration/slices/00-create.spec.js

```diff
@@ -12,10 +14,12 @@ describe("Create Slices", () => {
     cy
       .get('[data-cy=create-slice-modal]')
       .should('be.visible');
-    cy.get('input[data-cy=slice-name-input]').type('MySlice')
+
+    const name = randomPascalCase()
```

Comment:
Same, we have to avoid doing that, test should be determinist to ensure the protection.

Mutation testing are very specific tests that should be used with a proper tool in a very specific scope like a string formater function.

Reply:
It's more to do with the test failing if it's run more than once